.bazelci/presubmit.yml

```diff
@@ -1,5 +1,4 @@
 ---
-buildifier: true
```

Comment:
Leave a TODO to turn on again once we can disable specific warnings

Reply:
Done